doc/source/tune/examples/BUILD

```diff
@@ -17,28 +17,21 @@ py_test_run_all_notebooks(
     size = "medium",
     include = ["*.ipynb"],
     exclude = [
+        # TODO(krfricke): Enable this test for new persistence path.
         "bohb_example.ipynb",
```

Comment:
@krfricke could you take a look at this one later? I saw you moved these into a separate `new_storage` job, but I don't think those were being run.

I'm planning to merge this once all other tests pass so that we can get the other examples running on the new persistence path, and we can enable the bohb one as a follow up.